src/pages/api/locale/get-locale.ts

```diff
@@ -0,0 +1,81 @@
+import { Endpoint, getServerSideCookies, HttpStatus } from "@fwl/web";
```

Comment:
You should rename this file `index.ts` and declare the two locale handlers here, to follow our other endpoints. 🙂

Reply:
Good point. 😮